### ISSUE TITLE
Embed course information in section

### ISSUE THREAD

**Issue by seveibar** [rcos/venue#326]
Sections are not allowed to change courses, and courses rarely change major properties, so Sections should have some course information embedded between them to reduce db queries.

This will probably mean that many places that have withCourses can have that flag removed.

**Comment by kmcnellis** [comment]
I don't think we will want to implement this unless we can show it gives a peformace boost

**Comment by seveibar** [comment]
agreed, its sort of a mongodb convention to embed information though- im torn on whether or not its a wontfix

**Comment by kmcnellis** [comment]
We should get some server-side telemetry and monitor it
